### PR TITLE
BLT-18 upgrade node5 npm3

### PR DESCRIPTION
**Pull request by darrenbelding**
- Upgraded all dev dependencies
- Updated eslintrc to reflect eslint upgrade
- Removed combine-mq plugin. This is not currently compatible with npm3
- Added nvmrc file for nvm command usage
- Set engine-strict to fail package installs on incompatible node versions
- Ignore npm debug logs

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 rules:
-    indent: 4
+    "indent": 2
```
Comment (punkstar) [inline .eslintrc]: How come only this key is quoted?

Comment (darrenbelding) [inline .eslintrc]: Grabbed this from the eslint docs. Some of their rules show example usage with quotes, some not. I have just tested this locally and it doesn't appear that quoting effects anything. I've unquoted this to be consistent.

Comment (tgerulaitis) [inline .eslintrc]: It's YAML. YAML is happy with either.

**Comment by punkstar** [comment]
:+1:

**Comment by bobbyshaw** [comment]
When you said BLT in stand up I thought you'd had a particularly noteworthy sandwich :laughing:

**Pull request merged**